<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileMoveDemo</class>
 <widget class="QMainWindow" name="fileMoveDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fileMoveDemo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Lift">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>381</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>391</width>
             <height>361</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Right">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>361</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="Widget_ButtonGroup" native="true">
         <property name="minimumSize">
          <size>
           <width>621</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置路径</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action1">
   <property name="text">
    <string>设置图片路径</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
